lib/pbench/test/unit/agent/test_tool_meister.py

```diff
@@ -310,21 +309,29 @@ def setup_wait_for_process_tool(logger_name: str) -> Tool:
             (2, None, 1, 42, "5"),
             # 7 waits, 5 log entries
             (7, None, 5, None, "25"),
+            # 1 wait, 0 log entries
+            (1, "ctx-a", 0, 42, None),
+            # 2 waits, 1 log entry
+            (2, "ctx-b", 1, 42, "5"),
+            # 7 waits, 5 log entries
+            (7, "ctx-c", 5, None, "25"),
```

Comment:
Nevermind now, but it occurs to me that a better way to have done this would have been with nested `parameterize()`'s, one supplying the `ctx_name` and other providing the `waits` et al. (although, that would not have allowed for three unique values for `ctx_name`, but I'm not sure that that's important).